docs/src/main/asciidoc/building-native-image.adoc

```diff
@@ -348,12 +348,13 @@ In this guide we will use the first stage to generate the native executable usin
 ----
 ## Stage 1 : build with maven builder image with native capabilities
 FROM quay.io/quarkus/centos-quarkus-maven:{graalvm-version} AS build
-COPY src /usr/src/app/src
-COPY pom.xml /usr/src/app
+COPY pom.xml /project/usr/src/app/
+RUN mvn -f /project/usr/src/app/pom.xml -B de.qaware.maven:go-offline-maven-plugin:1.2.4:resolve-dependencies
+COPY src /project/usr/src/app/src
 USER root
-RUN chown -R quarkus /usr/src/app
+RUN chown -R quarkus /project/usr/src/app
```

Comment:
Why this change? I don't see how it's related to the other part

Reply:
Sorry for the delay, I was out of office for several weeks. You are correct, probably it's not really related. But without this change I wasn't able to build the final image. Maybe there was a change in the `quay.io/quarkus/centos-quarkus-maven` image.